scripts/components/collapsable/collapsable.scss

```diff
@@ -0,0 +1,41 @@
+.es-collapsable {
+	border-radius: 0.5rem;
+	box-shadow: 0 0 0 0 transparent;
+
+	transition: {
+		property: box-shadow, padding, margin;
+		timing-function: ease-out;
+		duration: 0.2s;
+	}
+
+	&__trigger {
+		display: grid;
+		align-items: center;
+		grid-template-columns: 1fr auto;
+		grid-template-rows: auto;
+		gap: 1rem;
+
+		border-bottom: 1px solid transparent;
+
+		transition: {
+			property: border-color;
+			timing-function: ease-out;
+			duration: 0.3s;
+		}
+  }
+
+	&.is-open {
+		box-shadow: 0 0 0 1px var(--wp-admin-theme-color, #194BD4);
+		padding: 0.5rem;
+
+		margin-left: -0.5rem;
+		margin-right: -0.5rem;
+
+    .es-collapsable__trigger {
```

Comment:
Why is this separate from the `&__trigger` above?

Reply:
This is taken from `CollapsableComponentUseToggle` styles, and the reasoning behind this is that these styles should only get applied to this selector: `.es-collapsable.is-open .es-collapsable__trigger`.

I can write it down using `$this`. Not sure if any alternatives come to your mind.
```scss
&.is-open {
  // ...
  #{$this}__trigger { 
    // ... 
  }
}
```

Reply:
Ah nevermind, the indents confused me, all good! 😄